spec/cypress/integration/cookiesBanner_spec.js

```diff
@@ -0,0 +1,29 @@
+describe("cookiesBanner", () => {
+
+  beforeEach(() => {
+    cy.visit('/');
+  });
+
+  it("displays cookies consent banner when user hasn't accepted yet", () => {
+    cy.get('.cookies-banner').should('be.visible');
+  });
+
+  it("hides cookies consent banner when user clicks accept button", () => {
+    cy.get('.cookies-banner__btn').click();
+
+    cy.get('.cookies-banner').should('not.be.visible');
+  });
+
+  it("displays cookies consent banner when user goes to a different page", () => {
+    cy.contains('Sign up').click();
+
+    cy.get('.cookies-banner').should('be.visible');
+  });
+
+  it("hides the cookies consent banner when they have already been accepted", () => {
+    cy.setCookie('allow_cookies', 'allowed');
```

Comment:
Instead of setting the cookies as `allowed` manually, is it possible to do it like this?

clicking the button -> reloading -> checking the banner is not visible

The reason for that is because e2e tests should behave just like the user does, and users wouldn't set their cookies manually.